va/va.h

```diff
@@ -2350,6 +2351,8 @@ typedef struct _VAPictureParameterBufferH264
         unsigned int value;
     } pic_fields;
     unsigned short frame_num;
+    unsigned char num_ref_idx_l0_default_active_minus1;
+    unsigned char num_ref_idx_l1_default_active_minus1;
```

Comment:
@xhaihao 
> The two flags are added for slice decoding base mode (VA_DEC_SLICE_MODE_BASE).

Can you explain what this mode does?

I don't believe these values can be used for anything unless you want to decode the whole slice header, since they can always take any value (including ones which would be invalid for num_ref_idx_lN_active_minus) and then be overridden.  But, there isn't enough information here to decode the whole slice header.